pysqldb3/pysqldb3.py

```diff
@@ -1414,7 +1404,7 @@ def _bulk_file_to_table(self, input_file=None, schema=None, table=None, table_sc
 
                 # Query one row to get columns
                 self.query("select {sq} * from {s}.stg_{t} {p}".format(s=schema, t=table, sq=sq, p=p), strict=False,
-                           timeme=False)
+                           timeme=False, internal = True)
 
                 column_names = self.queries[-1].data_columns
```

Comment:
this is still and issue `column_names = self.queries[-1].data_columns` will not return the correct results since the wuery is set to internal

Reply:
I thought I had reversed it. Hopefully the next PR refresh, you can see that I removed the `internal = True` label 
![image](https://github.com/user-attachments/assets/fcb932fa-9652-4749-8286-895694b12d4c)

Reply:
thats not the issue i was highlighting, i think it should be internal, but you need to update line 1410 since it wont be stored in the queries attribute